app/src/main/res/layout/activity_example1.xml

```diff
@@ -16,7 +16,6 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="Click"
-        android:textAppearance="@style/Roboto_Regular"
```

Comment:
I think it'd be good to leave something with this functionality in place using the new font apis, for demo purposes.